src/test/java/com/epam/qalab/SearchFieldTest.java

```diff
@@ -5,72 +5,63 @@
 created on 2/25/22   
 */
 
+import com.epam.qalab.pageobject.pages.CorrectSearchResultPage;
+import com.epam.qalab.pageobject.pages.IncorrectSearchResultPage;
+import com.epam.qalab.pageobject.pages.SearchResultPage;
+import com.epam.qalab.pageobject.pages.StartPage;
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import java.time.Duration;
 import java.util.List;
 
+import static org.openqa.selenium.support.locators.RelativeLocator.with;
 
-public class SearchFieldTest {
-    WebDriver driver;
-    String exec = "src/test/resources/chromedriver";
-    WebElement searchField;
-    String incorrect = "qwe4t5et6789tz3js4jdm1234567890";
-    String correct = "laptop";
+
+public class SearchFieldTest extends BaseTest {
 
     @BeforeTest
-    public void setUp() {
-        System.setProperty("webdriver.chrome.driver", exec);
-        driver = new ChromeDriver();
-        driver.get("https://www.amazon.com/");
-        driver.manage().window().maximize();
+    @Override
```

Comment:
There is probably no need to override this method as long as you don't change its behavior.